pkg/exasol-rest-api/properties_file_parser.go

```diff
@@ -0,0 +1,45 @@
+package exasol_rest_api
+
+import (
+	"errors"
+	"gopkg.in/yaml.v3"
+	"os"
+)
+
+func getPropertiesFromFile(filepath string, properties *ApplicationProperties) error {
+	propertiesFile, err := openFile(filepath)
+	if err != nil {
+		return err
+	}
+	err = decodePropertiesFile(propertiesFile, properties)
+	if err != nil {
+		return err
+	}
```

Comment:
I think we should close the file in case of an error. You could use `defer` (https://go.dev/blog/defer-panic-and-recover)

Reply:
fixed